test/api/search_controller_test.rb

```diff
@@ -105,14 +108,9 @@ class SearchControllerTest < ActionDispatch::IntegrationTest
 
     study = Study.find_by(name: "Test Study #{@random_seed}")
     facets = SearchFacet.where(data_type: 'string')
-    # format facet query string; this will be done by the search UI in production
-    facet_queries = []
-    facets.each do |facet|
-      if facet.filters.any?
-        facet_queries << [facet.identifier, facet.filters.map {|f| f[:id]}.join(',')]
-      end
-    end
-    facet_query = facet_queries.map {|query| query.join(':')}.join('+')
+
+    # find all human studies from metadata
+    facet_query = "species:#{HOMO_SAPIENS_FILTER[:id]}"
```

Comment:
I just realized this is wrong; it should read 

`"species:#{HOMO_SAPIENS_FILTER[:id]}+disease:#{NO_DISEASE_FILTER[:id]}"`